apps/zotonic_mod_acl_user_groups/src/mod_acl_user_groups.app.src

```diff
@@ -0,0 +1,11 @@
+{application, zotonic_mod_acl_user_groups, [
+    {description, "Zotonic module for organizing users into hierarchical groups"},
+    {vsn, "git"},
```

Comment:
By using ``git`` instead of a hard-coded version number that would then need to be replaced (or upped), we can just iterate through all modules on Travis and do `rebar3 hex publish` when Zotonic is tagged. Rebar will then use the Git tag number for the package’s version.